### ISSUE TITLE
Lambda() function creates Tensor, not Layer object

### ISSUE THREAD

**Issue by moonman239** [keras-team/keras#13296]
<em>Please make sure that this is a Bug or a Feature Request and provide all applicable information asked by the template.
If your issue is an **implementation question**, please ask your question on [StackOverflow](http://stackoverflow.com/questions/tagged/keras) or [on the Keras Slack channel](https://keras-slack-autojoin.herokuapp.com/) instead of opening a GitHub issue.</em>  

**System information**  
- Have I written custom code (as opposed to using example directory):
Yes.
- OS Platform and Distribution (e.g., Linux Ubuntu 16.04):
This happens on both Google Colab and on Mac OS X 10.14.6
- TensorFlow backend (yes / no):
Yes
- TensorFlow version:
v1.14.0-rc1-22-gaf24dc91b5 1.14.0
- Keras version:
2.2.5
- Python version:
3.6?
- CUDA/cuDNN version:
Not Available
- GPU model and memory:  
Not Available
You can obtain the TensorFlow version with:  
python -c "import tensorflow as tf; print(tf.GIT_VERSION, tf.VERSION)"  
You can obtain the Keras version with:  
python -c 'import keras as k; print(k.__version__)'  

**Describe the current behavior**  
Calling Lambda function with an Input tensor returns a Tensor.
**Describe the expected behavior**  
Per the documentation, calling Lambda with an Input tensor should return a Lambda object:

> Lambda
> `keras.layers.Lambda(function, output_shape=None, mask=None, arguments=None)`
> Wraps arbitrary expression as a Layer object.

**Code to reproduce the issue**  
```
from keras.layers import Input,Lambda
input_ = Input(shape=(1,))
lambda_ = Lambda(lambda x: x**2)(input_)
print(type(lambda_))
```
**Other info / logs**  
The "print" statement prints:
> <class 'tensorflow.python.framework.ops.Tensor'>
